csharp/Facebook.Yoga.Universal.sln

```diff
@@ -0,0 +1,112 @@
+﻿
```

Comment:
Only difference from the Facebook.Yoga.Desktop solution and this one is that the Universal tests are included here and the desktop tests are not.